about.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header>
+      <div class="flex nav-bar">
+        <a href="index.html"><img src="./images/gtco_logo2.png" alt="Logo" /></a>
+        <ul class="flex nav-list-item">
+          <li class="nav-item nav-about-page"><a href="about.html">About</a></li>
```

Comment:
- [ ] [optional] It would be nice if you changed this link from about to home page link since you already in the about page